### ISSUE TITLE
Adding getWeatherCondition topic fails

### ISSUE THREAD

**Issue by afestbaum** [arachnetech/homebridge-mqttthing#281]
Adding the _getWeatherCondition_ topic to an weatherstation accessory fails with the following error:

_Exception while creating services: TypeError: Cannot read property 'on' of undefined_

Looking at the code I think this is due to a mismatch in characteristics types between mqttthing and homebridge-lib.

```
function characteristic_WeatherCondition(service) {
        service.addOptionalCharacteristic(Eve.Characteristics.WeatherCondition); // to avoid warnings
        stringCharacteristic(service, 'weatherCondition', Eve.Characteristics.WeatherCondition, null, config.topics.getWeatherCondition, '-' );
    }
```

Here it uses the `Eve.Characteristics.WeatherCondition` type but this is defined as just Condition within homebridge-lib.

Could this be it?

**Comment by afestbaum** [comment]
https://github.com/ebaauw/homebridge-lib/commit/e92a2360fb5e2e7c83ba6ed3e0a90fd2aebf8e54#diff-ac8ca963e9a6af08492026d6d1f2f060

this is the commit where the rename happened

**Comment by arachnetech** [comment]
Thanks. Does sound as if we need to rename correspondingly in mqttthing.

**Comment by arachnetech** [comment]
What version of fakegato-history are you using? I'm using 0.5.6, which appears to be the latest (https://www.npmjs.com/package/fakegato-history) and I'm not seeing your issue.

**Comment by arachnetech** [comment]
OK, reproduced with homebridge-lib 4.7.7. I'd been using an earlier version.

**Comment by arachnetech** [comment]
Fixed in Version 1.1.15
